tests/beacon/test_helpers.py

```diff
@@ -707,6 +722,224 @@ def test_get_domain(pre_fork_version,
+    poc_1_keys = get_pubkey_for_indices(genesis_validators, proof_of_custody_1_indices)
+
+    assert bls.aggregate_pubkeys(poc_0_keys) == poc_0_key
+    assert bls.aggregate_pubkeys(poc_1_keys) == poc_1_key
+
+
+@given(st.data())
+def test_verify_vote_count(max_casper_votes, sample_slashable_vote_data_params, data):
+    (indices, some_index) = _list_and_index(data, max_size=max_casper_votes)
+    proof_of_custody_0_indices = indices[:some_index]
+    proof_of_custody_1_indices = indices[some_index:]
+
+    key = "aggregate_signature_poc_0_indices"
+    sample_slashable_vote_data_params[key] = proof_of_custody_0_indices
+    key = "aggregate_signature_poc_1_indices"
+    sample_slashable_vote_data_params[key] = proof_of_custody_1_indices
+
+    votes = SlashableVoteData(**sample_slashable_vote_data_params)
+
+    assert verify_vote_count(votes, max_casper_votes)
+
+
+def _select_indices(max, count):
+    '''
+    Randomly select some validator indices
+    '''
+    indices = []
+    for i in range(count):
+        next_index = random.randint(0, max - 1)
+        while next_index in indices:
+            next_index = random.randint(0, max - 1)
+        indices.append(next_index)
+    return indices
+
+
+def _get_indices_and_signatures(num_validators,
+                                num_indices,
+                                validators,
+                                message,
+                                privkeys):
+    indices = _select_indices(num_validators, num_indices)
+    privkeys = [privkeys[i] for i in indices]
+    domain = SignatureDomain.DOMAIN_ATTESTATION
+    signatures = tuple(
+        map(lambda key: bls.sign(message, key, domain), privkeys)
+    )
+    return (indices, signatures)
+
+
+def _correct_slashable_vote_data_params(params, validators, messages, privkeys):
+    valid_params = copy.deepcopy(params)
+
+    num_validators = len(validators)
+    num_indices = 5
+
+    key = "aggregate_signature_poc_0_indices"
+    (poc_0_indices, poc_0_signatures) = _get_indices_and_signatures(num_validators,
+                                                                    num_indices,
+                                                                    validators,
+                                                                    messages[0],
+                                                                    privkeys)
+    valid_params[key] = poc_0_indices
+
+    key = "aggregate_signature_poc_1_indices"
+    # NOTE: does not guarantee non-empty intersection
+    (poc_1_indices, poc_1_signatures) = _get_indices_and_signatures(num_validators,
+                                                                    num_indices,
+                                                                    validators,
+                                                                    messages[1],
+                                                                    privkeys)
```

Comment:
nitpick of formatting: https://github.com/ethereum/snake-charmers-tactical-manual/blob/master/style-guide.md#closing-parenthesisbracketbrace